src/components/skip-link/skip-link.yaml

```diff
@@ -1,3 +1,25 @@
+params:
+- name: text
+  type: string
+  required: true
```

Comment:
text and html are not required in the current table, since they have a default

Reply:
Think I was thinking of the back-link, it probably should have a default... Disregard this comment :)